k8s/k8s.go

```diff
@@ -316,6 +445,29 @@ func overrideIPAMResult(ipAddrsNoIpam string, logger *log.Entry) (*types.Result,
 	return &result, nil
 }
 
+// parseIPAddrs is a utility function that parses string of IPs in json format that are
+// passed in as a string and returns a slice of string with IPs.
+// It also makes sure the slice isn't empty.
+func parseIPAddrs(annotation, ipAddrsStr string, logger *log.Entry) ([]string, error) {
```

Comment:
Rather than pass the annotation in here just for logging purposes, I'd much rather do the context logging outside of this function.